Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM tomcat:8-jre8-temurin
+
+ENV PATH="/usr/local/bin/liquibase:${PATH}" 
+
+COPY docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+COPY ./waltz-data/src/main/ddl/liquibase/*.xml /opt/waltz/liquibase/
+COPY ./waltz-web/target/waltz-web.war /usr/local/tomcat/webapps
```

Comment:
suggestion: copy the war file as `ROOT.war` so that waltz runs on `/` instead of `/waltz-web`